src/alfasim_sdk/testing/fixtures.py

```diff
@@ -120,17 +120,19 @@ def run(self, *, number_of_threads: int = -1) -> Results:
         self.alfacase_data_folder.mkdir()
         self.alfacase_file.write_text(alfacase, encoding="utf-8")
 
-        self._run_simulator(
-            self.get_runner()
-            + [
-                "--run-simulation",
-                f"--alfacase-file={str(self.alfacase_file)}",
-                f"--number-of-threads={number_of_threads}",
-            ],
-            cwd=project_folder,
-        )
+        try:
```

Comment:
Doubt:
I know that a generic error would be raised. But what is the name of this error? Which one would be raised?

Reply:
`subprocess.CalledProcessError`